types/pino/index.d.ts

```diff
@@ -227,6 +227,10 @@ declare namespace P {
         /**
          * Changes the property `level` to any string value you pass in. Default: 'level'
          */
+        levelKey?: string;
+        /**
+         * (DEPRECATED, use `levelKey`) Changes the property `level` to any string value you pass in. Default: 'level'
```

Comment:
Some libraries use the JSDoc `@deprecated` syntax, which I believe has some tool support.  I don't think there's official guidance either way, but I thought I'd mention the option.